routing/routing.xml

```diff
@@ -614,12 +630,22 @@
 			<select value="1.3" t="highway" v="steps"/>
 			<select value="3" t="route"   v="ferry"/>
 			<select value="5" t="bicycle" v="dismount"/>
-			<select value="22" t="bicycle" v="designated"/>
-			<if param="avoid_unpaved">
-				<select value="4" t="highway" v="track"/>
-				<select value="4" t="highway" v="footway"/>
-				<select value="3" t="highway" v="path"/>
-			</if>
+			<!-- <select value="22" t="bicycle" v="designated"/>-->
+			<!-- Line above: Does the "bicycle designated" work here?
+			So far, we have assigned a kmh value to tracks, paths etc - 
+			can a kmh value also be assigned to a tag like "bicycle designated"?
+			If so: Would all tracks, paths etc tagged with "bicycle designated"
+			be multiplied later on (so that, if they are tagged with "paved" or
+			"asphalt", the 23 kmh would be multiplied with the "paved" factor)?
+			Could someone please help me understand this?
+			If it works as described, it adds some great possibilities.-->
+
+			<!-- <select value="4" t="highway" v="track"/> <select value="4" t="highway" v="footway"/> <select value="3" t="highway" v="path"/>-->
+			<!-- Line above: This line contained downgrades for the
+			"avoid unpaved" mode and has been moved further down.
+			Osmand first has to check if a path or track is paved,
+			asphalt, etc. Only if no tag like "asphalt" has been
+			found, then paths / tracks etc should be downgraded.-->
```

Comment:
I did want to suggest to delete it - just suggested to move it further down. 

Once we have checked that there is no asphalt etc, the paths, tracks, footways will be heavily downgraded in "unpaved" mode (please look further down - I think it makes sense there).